engine.go

```diff
@@ -147,6 +147,9 @@ func (e *engine) render() {
 		case RPrompt:
 			e.rprompt = e.renderBlockSegments(block)
 		}
+		if block.VerticalOffset != 0 {
```

Comment:
Isn't this one doing it twice?

Reply:
I'll revert my `vertical_offset` changes in this PR.

But to answer your question, this was actually undoing the vertical_offset, notice the `-` in the `e.renderer.changeLine(-block.VerticalOffset)`. I noticed `vertical_offset` had some buggy feeling behavior depending on how and where you use it, and this was an attempt to make it feel a little more robust. I think that probably deserves it's own PR, as it's a pretty nuanced issue, and not really related to rprompt.  Also I don't think my changes fully fixed it.

Reply:
this has been reverted